config/foreman.hiera/family/RedHat.yaml

```
@@ -1,3 +1,5 @@
 ---
 # Disable additional Phusion Passenger repo on EL, preferring EPEL/Foreman's
 apache::mod::passenger::manage_repo: false
+redis::params::default_install: false
+redis::params::manage_repo: false
```

This should be set on the redis class itself
```suggestion
redis::manage_repo: false
```